### PR TITLE
Sacrificial lamb joker

### PR DESCRIPTION
**Pull request by BBBalatroMod**
sacrificial_lamb.lua in the joker folder and edited paperback.lua to enable it

### CODE REVIEW

File: joker/sacrificial_lamb.lua [hunk 1]
```diff
@@ -0,0 +1,45 @@
+SMODS.Joker {
+    key = 'sacrificial_lamb',
+    loc_txt = {
+        name = "Sacrificial Lamb",
+        text = {
+            "Gains {C:mult}+5 Mult{} when",
+            "for each card {C:attention}sold{}",
+            "{C:inactive}(Currently gives {}{C:mult}+#1#{} {C:inactive}Mult){}"
```
Comment (OppositeWolf770) [inline joker/sacrificial_lamb.lua:8]: This effect should be on every card "destroyed", not sold

File: joker/sacrificial_lamb.lua [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+    end,
+
+    loc_vars = function(self, info_queue, center)
+        return { vars = {center.ability.extra.mult, center.ability.extra.x_mult} }
+    end,
+    
+    calculate = function(self, card, context)
+        if context.selling_card and not context.blueprint then
+            card.ability.extra.mult = card.ability.extra.mult + card.ability.extra.mult_mod
+        end
```
Comment (OppositeWolf770) [inline joker/sacrificial_lamb.lua:34]: Similar to above, the context needs to be when a card is destroyed, not sold. Although I'm not too sure what that context is

**Review by OppositeWolf770 — CHANGES_REQUESTED** [review]

**Comment by BBBalatroMod** [comment]
I mostly just used the context and description from campfire as its effect is pretty close to this one, but i'll try to find a context that applies to destroyed cards

File: joker/sacrificial_lamb.lua (hunk 2 above)
Comment (GitNether) [reply joker/sacrificial_lamb.lua:34]: `remove_playing_cards` works for playing cards beind destroyed, idk about Jokers

File: joker/sacrificial_lamb.lua (hunk 1 above)
Comment (GitNether) [reply joker/sacrificial_lamb.lua]: - "when for each card sold" grammar aint grammar'ing
- As a tip, take a look at other jokers already in the game to get a more natural wording of the functionality of the joker.
Namely, adapting from campfire:

This Joker gains +5 Mult for each card or Joker destroyed

This also includes the highlighting of the important bits, of course.

**Review by GitNether — CHANGES_REQUESTED** [review]
I think "or Joker" makes more sense. Also capitalizing the word Joker

**Comment by GitNether** [comment]
Is this ever getting fixed? lol

**Comment by OppositeWolf770** [comment]
@BBBalatroMod I can work on this and get it finished if you are busy and can't work on it

File: joker/sacrificial_lamb.lua (hunk 2 above)
Comment (OppositeWolf770) [reply joker/sacrificial_lamb.lua]: For Jokers, I had to implement a brand new context.

**Review by OppositeWolf770 — APPROVED** [review]

**Review by OppositeWolf770 — APPROVED** [review]

**Review by OppositeWolf770 — APPROVED** [review]

**Review by OppositeWolf770 — APPROVED** [review]

**Pull request merged**